src/ngx_stream_upsync_module.c

```diff
@@ -1155,7 +1155,7 @@ ngx_stream_upsync_consul_parse_json(void *data)
         cJSON *temp1 = cJSON_GetObjectItem(server_next, "Key");
         if (temp1 != NULL && temp1->valuestring != NULL) {
             p = (u_char *)ngx_strrchr(temp1->valuestring, '/');
-            if (ngx_stream_upsync_check_key(p) != NGX_OK) {
```

Comment:
Like: 
if (p == NULL) {
     ngx_log_error(NGX_LOG_ERR, ngx_cycle->log, 0, "upsync_parse_json: %s format is illege", 
                             temp1->valuestring)
}
or something.

Are you interested to make a pr again? thanks.